src/Xpress.jl

```diff
@@ -92,7 +92,15 @@ module Xpress
 
         # some lics require special check
         # -------------------------------
-        userlic()
+        xpauth_path = ""
+        if is_unix()
+          if haskey(ENV, "XPRESSDIR")
+            xpauth_path = joinpath(ENV["XPRESSDIR"], "bin")
+          else
+            xpauth_path = joinpath(dirname(dirname(xprs)), "bin")
+          end
+        end
+        userlic(xpauth_path=xpauth_path)

```

Comment:
But, I think we should move this code to automatically obtaining `xpauth_variable` inside the `userlic()` function. 
I am considering removing `__init__()` soon, because JuMP used to load solvers automatically, but now solvers will have to be included explicitly. So it would be reasonable to user use `Env()` and `userlic()` explicitly also.